src/styles/_catalog.scss

```diff
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+    transition-property: all;
```

Comment:
Sorry, but that is bad practice to use all for transition-property)

Reply:
@OlehMy I agree, that's what Yura said yesterday, I paid attention,  changed it